style.css

```diff
@@ -359,43 +393,73 @@ img {vertical-align: middle;}
 }
 
 .w_t_b_text{
-	color: white;
-	font-family: sans-serif;
 	position: absolute;
-	top: 10%;
+	top: 18%;
 	left: 15%;
 	transform: translate(-10%,-10%);
-	font-size: 15px;
 	text-align: center;
-	height: 100%;
+	color: white;
+	line-height: 22px;
+	font-family: 'Roboto', sans-serif;
+	font-weight: 400;
+}
+
+.w_t_b_text h1{
+	font-size: 24px;
+}
+
+.w_t_b_text p{
+	font-size: 16px;
+}
+
+.w_t_b_img h2{
+	font-family: 'Open Sans', sans-serif;
+	font-weight: 300;
 }
 
+
 .text{
 	color: white;
-	font-family: sans-serif;
 	position: absolute;
 	top: 7%;
 	left: 15%;
 	transform: translate(-10%,-10%);
-	font-size: 15px;
 	text-align: center;
+	font-weight: 400;
+}
+
+.text h1{
+	font-family: 'Open Sans', sans-serif;
+	font-size: 23px;
 }
 
+.text h2{
+	margin-top: -15px;
+	font-size: 15px;
+	font-family: 'Open Sans', sans-serif;
+	font-style: italic;
+}
+
+.text p{
+	margin-top: -5px;
+	font-family: 'Open Sans', sans-serif;
+	font-size: 14px;
+}
 
 .overlayLeft{

```

Comment:
Overall, I think the dark orange, and bright orange for the overlay background colour may need to be changed for readability, as well as the light blue, a gradient might be a good idea.